eoscostarica/guide.eoscostarica.io
# feat(frontend): Improved footer styles #471

### Improve footer styles

### What does this PR do?

- resolves #471 
Note: react-icons was installed: npm install react-icons --save or yarn install react-icons

### Steps to test
1. Navigate to the bottom of [guide.eosio](https://guide.eoscostarica.io/) and see page footer

#### CheckList
